renderer/pages/settings.js

```diff
@@ -178,36 +178,51 @@ class Settings extends React.Component {
             </TopBar>
             <Container pt={3}>
               <Heading h={4}><FormattedMessage id='Settings.Sharing.Label' /></Heading>
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.UploadResults' />}
-                optionKey='sharing.upload_results'
-                config={config} />
-
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.IncludeNetwork' />}
-                optionKey='sharing.include_asn'
-                config={config} />
-
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.IncludeCountryCode' />}
-                optionKey='sharing.include_country'
-                config={config} />
-
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.IncludeIP' />}
-                optionKey='sharing.include_ip'
-                config={config} />
-
-              <NumberOption
-                onConfigSet={this.reloadConfig}
-                label={<Text>Websites tested (0 means all)</Text>}
-                optionKey='nettests.websites_url_limit'
-                config={config} />
-              <Text pt={3}>OONI Probe Desktop v{pkgJson.version}</Text>
+              <Flex flexDirection='column'>
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.UploadResults' />}
+                  optionKey='sharing.upload_results'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.IncludeNetwork' />}
+                  optionKey='sharing.include_asn'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.IncludeCountryCode' />}
+                  optionKey='sharing.include_country'
+                  config={config}
```

Comment:
While we are at it, let's also remove the ability to disable submitting the country code. See: https://github.com/ooni/probe/issues/1105

Reply:
removed